examples/contracts/DAAppNFT/src/Contract.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.10;
+
+import "solmate/tokens/ERC721.sol";
+import "openzeppelin-contracts/contracts/utils/Strings.sol";
+import "openzeppelin-contracts/contracts/access/Ownable.sol";
+
+error MintPriceNotPaid();
+error MaxSupply();
+error NonExistentTokenURI();
+error WithdrawTransfer();
```

Comment:
oh never seen errors defined outside of the contract like this, usually I see the error definitions in the scope of the contract definitions

Reply:
Hmm I was copying how the tutorial was defining errors lol I still think this is functionally correct, and it's nice to call them up front at the top of the contract IMO.